components/brave_ads/browser/ads_tab_helper.cc

```diff
@@ -76,29 +96,34 @@ void AdsTabHelper::DocumentOnLoadCompletedInMainFrame() {
 
   content::RenderFrameHost* render_frame_host =
       source_page_handle->web_contents()->GetMainFrame();
+
+  RunIsolatedJavaScript(render_frame_host);
+}
+
+void AdsTabHelper::RunIsolatedJavaScript(
+    content::RenderFrameHost* render_frame_host) {
   DCHECK(render_frame_host);
 
   dom_distiller::RunIsolatedJavaScript(render_frame_host,
       "document.body.innerText",
-          base::BindOnce(&AdsTabHelper::OnWebContentsDistillationDone,
-              weak_factory_.GetWeakPtr(),
-                  source_page_handle->web_contents()->GetLastCommittedURL(),
-                      base::TimeTicks::Now()));
+          base::BindOnce(&AdsTabHelper::OnJavaScriptResult,
+              weak_factory_.GetWeakPtr()));
```

Comment:
this is wrong spacing, can be fixed with `clang-format`

Reply:
Raised https://github.com/brave/brave-browser/issues/11066 to fix this globally through `brave_ads`, `bat-native-ads` and `bat-native-usermodel`